2-mandatory/4-stocks-V2-with-arrow-Functions.js

```diff
@@ -0,0 +1,89 @@
+/*
+    THESE EXERCISES ARE QUITE HARD. JUST DO YOUR BEST, AND COME WITH QUESTIONS IF YOU GET STUCK :)
+
+    Imagine we a working for a finance company. Below we have:
+        - an array of stock tickers
+        - an array of arrays containing the closing price for each stock in each of the last 5 days.
+            For example, CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[2] contains the prices for the last 5 days for STOCKS[2] (which is amzn)
+*/
+
+/* ======= Stock data - DO NOT MODIFY ===== */
+const STOCKS = ['aapl', 'msft', 'amzn', 'googl', 'tsla'];
+
+const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
+  [179.19, 180.33, 176.28, 175.64, 172.99], // AAPL
+  [340.69, 342.45, 334.69, 333.20, 327.29], // MSFT
+  [3384.44, 3393.39, 3421.37, 3420.74, 3408.34], // AMZN
+  [2951.88, 2958.13, 2938.33, 2928.30, 2869.45], // GOOGL
+  [1101.30, 1093.94, 1067.00, 1008.87, 938.53], // TSLA
+];
+
+/*
+    We want to understand what the average price over the last 5 days for each stock is.
+    Implement the below function, which
+        - Takes this CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS array as input (remember, it's an array of arrays)
+        - Returns an array containing the average price over the last 5 days for each stock.
+            For example, the first element of the resulting array should contain Apple’s (aapl) average stock price for the last 5 days.
+            The second element should be Microsoft's (msft) average price, and so on.
+    The average value should be rounded to 2 decimal places, and should be a number (not a string)
+
+    Hint 1: To calculate the average of a set of values, you can add them together and divide by the number of values.
+        So the average of 5, 10 and 20 is (5 + 10 + 20) / 3 = 11.67
+    Hint 2: If the problem seems complex, try breaking it down into smaller problems.
+        Solve the smaller problems, and then build those solutions back up to solve the larger problem.
+        Functions can help with this!
+*/
+
+const getAveragePrices = (arr) => arr.map((e) => Math.round((e.reduce((a, i) => a + i, 0) / 5) * 100) / 100);
```

Comment:
Well done on using map and reduce.

You have hardcoded 5 which is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) and should be avoided. How do you change the code so that the magic number is removed?